app.js

```diff
@@ -408,11 +410,80 @@ client.on("message", async message => {
                 message.channel.send("Discord's best practices for bots state that failed commands should fail silently");
             }
         }
-
     };
     // message.delete();
     if (message.guild.id.toString().includes(config.logserver)) {
-        client.channels.get(config.logchannel).send( message.author.username + '(' + message.author.id + ') ran command `' + message.cleanContent + '` in ' + message.channel, {"split":true});
+        client.channels.get(config.logchannel).send({embed:{
+            color: colors.green,
+            author: {
+                name : message.author.username,
+                icon_url: message.author.displayAvatarURL
+            },
+            title: "Command ran in #".concat( message.channel.name ),
+            fields: [{
+                name: "Command",
+                value: "`".concat( message.cleanContent, "`")
+            }],
+            timestamp: new Date(),
+            footer: {
+                icon_url: client.user.displayAvatarURL,
+                text: "User ID: ".concat(message.author.id.toString())
+            }
+        }})
+    }
+});
+
+client.on("messageDelete", (message) => {
+    output = "Message: '".concat( message.cleanContent, "' by ", message.author.username, " (", message.author.id, ") in #", message.channel.name, " was deleted")
+    console.log(output)
+    client.channels.get(config.logchannel).send({embed: {
+        color: colors.red,
+        author: {
+            name: message.author.username,
+            icon_url: message.author.displayAvatarURL
+        },
+        title: "Message deleted in #".concat( message.channel.name ),
+        description: "The following message was deleted:",
+        fields: [{
+            name: "Message",
+            value: "`".concat(message.cleanContent, "`")
+        }],
+        timestamp: new Date(),
+        footer: {
+            icon_url: client.user.displayAvatarURL,
+            text: "User ID: ".concat(message.author.id.toString())
```

Comment:
when using the `+` with a string `.toString()` is automatically called on anything that isnt a string so it just becomes `text: "User ID: " + message.author.id`